src/components/accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react"
+import type { AccordionData } from "./Accordion"
+import clsx from "clsx"
+
+function AccordionItem({
+    data,
+    isOpen,
+    btnOnClick,
+    titleLabels
+}: {
+    data: AccordionData
+    isOpen: boolean
+    btnOnClick: () => void
+    titleLabels?: boolean
+}) {
+    const [ height, setHeight ] = useState("")
```

Comment:
I believe this state is unnecessary. Consider refactoring to a ternary expression instead, so simplify the code.

Reply:
Solved 🙂